cmd/pool/order.go

```diff
@@ -116,6 +116,18 @@ var sharedFlags = []cli.Flag{
 			channelTypePeerDependent,
 			channelTypeScriptEnforced),
 	},
+	cli.StringSliceFlag{
+		Name: "allowed_node_ids",
```

Comment:
I assume this is specified multiple times to add multiple IDs? Perhaps the flag name should be singular then and the description should say "can be specified multiple times".